draft-ietf-sframe-enc.md

```diff
@@ -391,7 +414,7 @@ aspects of the AEAD and the hash algorithm below:
   size of a "tag" that is added to the plaintext)
 
 * `AEAD.Nka` - For cipher suites using the compound AEAD described in
```

Comment:
I originally read `Nka` as "size-of-key-for-AEAD", which made sense to me, but then I realized that since we're always using AEAD, it's really "size-of-key-for-compound-AEAD".  Maybe `Nkc`?

Reply:
`Nke` looks good to me.